packages/web-components/src/wallet-connection/watchWallet.ts

```diff
@@ -67,19 +53,15 @@ export const watchWallet = (
   };
 
   const publicSubscriberPathsNotifierKit = makeNotifierKit(
-    /** @type {  import('@agoric/smart-wallet/src/smartWallet.js').CurrentWalletRecord['offerToPublicSubscriberPaths'] | null } */ (
-      null
-    ),
+    /** @type {  import('@agoric/smart-wallet/src/smartWallet.js').CurrentWalletRecord['offerToPublicSubscriberPaths'] | null } */ null,
   );
 
   const walletUpdatesNotifierKit = makeNotifierKit(
```

Comment:
Can we provide a type arg to `makeNotifierKit` instead of casting here?

Reply:
this was just a prettier change. I forgot to convert from jsdoc so TS isn't even seeing it. I'll fix